test-harness/README.md

```diff
@@ -0,0 +1,36 @@
+## Generating binaries
+
+For anything to work, the binaries have to be generated first with: `yarn build.binaries`. This will generate three binaries.
+
+### Alternatively, run any of the following:
+
+1. `yarn build.binary` will assume the host machine
+2. `HOST=host yarn build.binary` will generate a binary for the current host machine
+3. `HOST=node10-linux-x64 yarn build.binary` will generate a binary for node10-linux-x64
```

Comment:
We should probably tell the user what are the possible choices or at least put a link to where the choices are (`pkg` docs should be good).